src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit-property.element.ts

```diff
@@ -72,26 +119,42 @@ export class UmbDocumentTypeWorkspacePropertyElement extends UmbLitElement {
 		this.dispatchEvent(new CustomEvent('partial-property-update', { detail: partialObject }));
 	}
 
-	renderInheritedProperty() {
-		return this.property
-			? html`
-					<div id="header">
-						<b>${this.property.name}</b>
-						<i>${this.property.alias}</i>
-						<p>${this.property.description}</p>
-					</div>
-					<div id="editor"></div>
-			  `
-			: '';
-	}
-
 	@state()
 	private _aliasLocked = true;
 
 	#onToggleAliasLock() {
 		this._aliasLocked = !this._aliasLocked;
 	}
 
+	#requestRemove(e: Event) {
+		e.preventDefault();
+		e.stopImmediatePropagation();
+		if (!this.property || !this.property.id) return;
+
+		const Message: UmbConfirmModalData = {
+			headline: `${this.localize.term('actions_delete')} property`,
+			content: html`<umb-localize key="contentTypeEditor_confirmDeletePropertyMessage" .args=${[
+				this.property.name || this.property.id,
+			]}>
+					Are you sure you want to delete the property <strong>${this.property.name || this.property.id}</strong>
+				</umb-localize>
+				</div>`,
+			confirmLabel: this.localize.term('actions_delete'),
+			color: 'danger',
+		};
+
+		const modalHandler = this._modalManagerContext?.open(UMB_CONFIRM_MODAL, Message);
+
+		modalHandler
+			?.onSubmit()
+			.then(() => {
+				this.dispatchEvent(new CustomEvent('property-delete'));
+			})
+			.catch(() => {
```

Comment:
I don't think we need this catch part. So we can just remove it.

Reply:
When there is no catch, an error appears in the console when pressing cancel when trying to delete.

Reply:
It doesn't make an actual difference other than just removing the console error 😊

Reply:
Oh, okay. makes good sense. I'll have a look if we can avoid that.